<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
background-color : gray;
}</string>
    </property>
    <property name="title">
     <string>Методы</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>381</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="bnt_erase">
          <property name="text">
           <string>erase</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bnt_max_size">
          <property name="text">
           <string>maxsize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bnt_pop_back">
          <property name="text">
           <string>pop_back</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bnt_push_back">
          <property name="text">
           <string>push_back</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="bnt_reserve">
          <property name="text">
           <string>reserve</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bnt_resize">
          <property name="text">
           <string>resize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bnt_size">
          <property name="text">
           <string>size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bnt_capacity">
          <property name="text">
           <string>capacity</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="bnt_at">
          <property name="text">
           <string>at</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bnt_clear">
          <property name="text">
           <string>clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bnt_data">
          <property name="text">
           <string>data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bnt_empty">
          <property name="text">
           <string>empty</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="lbl_vector">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>811</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>[]</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_vvod">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ввод данных: </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>44</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Стартовое количество элементов: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spb_kolvo_elements">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>40</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="bnt_random">
       <property name="text">
        <string>Сгенерировать массив рандомных элементов</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
